support/mender-inventory-geo

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+#
+# Example script called by Mender client to collect inventory data for a
+# particular device. The script needs to be located in $(datadir)/mender and its
+# name shall start with `mender-inventory-` prefix. The script shall exit with
+# non-0 status on errors. In this case the agent will discard any output the
+# script may have produced.
+#
+# The script shall output inventory data in <key>=<value> format, one entry per
+# line. Entries appearing multiple times will be joined in a list under the same
+# key.
+#
+# $ ./mender-inventory-geo
+# country=Poland
+# city=Krakow
+# latitude=50.05750
+# longitude=19.98020
+# ipv4=127.0.0.1
+#
+# The example script collects information on geo localization
+# this can be done in one line:
+# curl https://ipvigilante.com/`curl -4 ifconfig.io 2>/dev/null` 2>/dev/null | jq -r 'select(.status == "success") | "country=" + .data.country_name, "city="+.data.city_name, "latitude="+.data.latitude, "longitude="+.data.longitude, "ipv4="+.data.ipv4 | @text'
+# but in the following we assume busybox version of the utilities.
+
+# the names of the attributes holding the localization data can be configured here
+ATTR_NAME_IP="a-ip"
+ATTR_NAME_CONTINENT="a-continent"
+ATTR_NAME_COUNTRY="a-country"
+ATTR_NAME_CITY="a-city"
+
+# this we will use for everything, wget in busybox does not support -O-
+tmpfile=`mktemp`
+[ -f "${tmpfile}" ] || exit 1
+# setup cleanup
+function geo_cleanup() {
+ rm -f "${tmpfile}"
+}
+
+trap geo_cleanup EXIT
+trap geo_cleanup SIGHUP
+trap geo_cleanup SIGQUIT
+trap geo_cleanup SIGINT
+trap geo_cleanup SIGTERM
+trap geo_cleanup SIGKILL
+
+# find the ip address
+wget -qO "${tmpfile}" --header "Content-Type: text/plain;" \
+     --header 'Host: ifconfig.io' --header 'User-Agent: curl/7.67.0' \
```

Comment:
So *wget* or *curl*?

Reply:
well, this is something I found works with ifconfig.io, does not matter much I guess if wget is pretending to be curl. it si on purpose basically.

Reply:
sure. moved before awk.